vonk/features/terminologyintegration.rst

```diff
@@ -0,0 +1,100 @@
+.. _feature_terminologyintegration:
+
+External terminology services
+=============================
+
+.. warning:: The TerminologyIntegration plugin does not work together with the Vonk Terminology plugin. 
+
+To be able to use more elaborate terminology support, 
+we provide the option to integrate external FHIR terminology services with Vonk. 
+
+Vonk can redirect the following terminology operations to external services:
+* $validate-code
+* $expand
+* $lookup
+* $translate
+* $subsumes
+
+To enable the TerminologyIntegration it needs to be added to the PipelineOptions in the appsettings.
+
+.. note::
+Adding Vonk.Plugins.TerminologyIntegration means you cannot use Vonk.Plugins.Terminology. Vonk will log a warning if you have both plugins included in the appsettings. 
+
+.. code-block:: JavaScript
+
+"PipelineOptions": {
+    "PluginDirectory": "./plugins",
+    "Branches": [
+      {
+        "Path": "/",
+        "Include": [
+          "Vonk.Core",
+          "Vonk.Fhir.R3",
+          "Vonk.Fhir.R4",
+          //"Vonk.Fhir.R5",
+          "Vonk.Repository.Sql.SqlVonkConfiguration",
+          "Vonk.Repository.Sqlite.SqliteVonkConfiguration",
+          "Vonk.Repository.MongoDb.MongoDbVonkConfiguration",
+          "Vonk.Repository.CosmosDb.CosmosDbVonkConfiguration",
+          "Vonk.Repository.Memory.MemoryVonkConfiguration",
+          "Vonk.Subscriptions",
+          "Vonk.Smart",
+          "Vonk.UI.Demo",
+          "Vonk.Plugin.DocumentOperation.DocumentOperationConfiguration",
+          "Vonk.Plugin.ConvertOperation.ConvertOperationConfiguration",
+          "Vonk.Plugin.BinaryWrapper",
+          "Vonk.Plugin.MappingToStructureMap.MappingToStructureMapConfiguration",
+          "Vonk.Plugin.TransformOperation.TransformOperationConfiguration",    
+          "Vonk.Plugin.Audit",          
+          "Vonk.Plugins.TerminologyIntegration",
+          "Vonk.Plugin.Mapping"          
+        ],
+        "Exclude": [
+          "Vonk.Subscriptions.Administration"
+        ]
+      }
```

Comment:
Missing a closing } for the PipelineOptions, or perhaps add an "... etc ..." to indicate there can be more paths. People tend to copy the text to their own settings...

Reply:
Perhaps we also need an example of excluding the normal Vonk Terminology plugin? Or is that not loaded by default? The note above seems to suggest we need to do something with that, but this doesn't really show me what changes were made or need to be made.
If you put the note below these settings, followed by an example of the exclusion, it would be more clear.